### ISSUE TITLE
Sessão Plenária: Oradores do Expediente

### ISSUE THREAD

**Issue by andreagudo** [interlegis/sapl#1872]
Limitar os parlamentares disponíveis para inserir na lista de Oradores do Expediente da sessão plenária. Devem aparecer apenas os ativos com mandato na legislatura em que a sessão está sendo inserida.
![captura de tela de 2018-04-16 07-59-57](https://user-images.githubusercontent.com/19169882/38805604-bd0cd12e-414c-11e8-97bf-6436890d7d2e.png)

**Comment by VictorFabreF** [comment]
Bom dia, 
Fiz uma verificação e notei que existem 22 parlamentares ativos na sua base. Alguns destes não possuem mandato na legislatura atual. Segue lista com estes parlamentares:

Danilo Hoffmann
Derli Duttel
Eldo Karsburg
Enar Ernst
Gerson Halberstadt
Hasso Bräunig
Hildemar Wappler
Laise Berger
Léo Gehrke
Liberto Boijink
Márcio Karsburg
Romeu Ehrhardt
Vilson Dias

Também os parlamentares João de Deus e Márcio Halberstadt constam como inativos na casa, mesmo tendo mandato na legislatura atual. Sugerimos que você edite os parlamentares supracitados para que conste na base somente aqueles ativos na atual legislatura.

**Comment by andreagudo** [comment]
João e Márcio devem continuar inativos, pois estão em licença; o ideal é que inativos não apareçam na lista para evitar eventuais equívocos. 2) Os ativos de ouras legislaturas estão sendo usados para o registro de sessões antigas; realmente, inativando-os, não apareceriam. De qualquer forma, no 2.5 tal lista contém apenas ativos da legislatura a que a sessão plenária pertence.

**Comment by edwardoliveira** [comment]
Não, **totalmente diferentes** as implementações. Aqui não dá mesmo. Sinto muito.

**Comment by andreagudo** [comment]
Para sessões antigas não funciona ainda. Aparecem para seleção os parlamentares ativos da atual legislatura, não os da legislatura correpondente.
https://sapl3.agudo.rs.leg.br/sessao/1855/oradorexpediente/create
![captura de tela de 2018-04-18 09-09-12](https://user-images.githubusercontent.com/19169882/38931023-35c3e408-42e8-11e8-80ee-f39560c885e1.png)
